docs/_plugins/search_index.rb

```diff
@@ -6,6 +6,8 @@
 require 'elasticsearch'
 require 'nokogiri'
 
+$outdated_editions = ['Spark NLP 2.1', 'Healthcare NLP 2.0']
```

Comment:
Why not `OUTDATED_EDITIONS`?

Reply:
Yeah I updated it.. I doesn't work with `outdated_editions` . But works with the Ruby constant `OUTDATED_EDITIONS`